lookout/style/format/rules.py

```diff
@@ -108,8 +108,8 @@ def rules(self) -> List[Rule]:
         return self._rules
 
     @property
-    def origin(self) -> Mapping[str, Any]:
-        return self._origin
+    def config(self) -> Mapping[str, Any]:
```

Comment:
the property doesn't seem necessary

Reply:
yes, but I prefer to keep it because `_config` should be read-only. The property implements exactly this. Also before we have `origin` with the same meaning.

Reply:
why not but we have plenty of readonly attributes in the codebase that are not `@property`'d. We should at least harmonize at some point.

Reply:
+1 for leaving the property and adding other readonly properties.

Reply:
AFAIK all params of trainable rules are read only for example. Same for many other params in many other places

Reply:
oh, of course I mean this class only. But I keep it as an issue. Better to have separate PR for that.